### ISSUE TITLE
open nav menus on menu hover 

### ISSUE THREAD

**Issue by eugenethreat** [phillycommunitywireless/phillycommunitywireless#134]
currently requires a click; think it would look slick for the menus to open on hover as well

**Comment by JoshMLeslie** [comment]
Hello, just joined this week. My 2 cents is that mobile and desktop experiences should be as similar as possible. To that, hover states should be avoided when possible, and especially hover as an action.

**Comment by eugenethreat** [comment]
I agree that hover states should be avoided on mobile, but on desktop they have a nice flair. This could definitely be improved ... I believe whoever implemented this just wrote a bunch of raw JS/jquery. Maybe a slider menu or something would look better. 

![image](https://github.com/user-attachments/assets/b4906e8c-b9a7-451c-9821-58461cdcb905)
